packages/venia-concept/src/components/Navigation/navHeader.js

```diff
@@ -18,13 +18,17 @@ class NavHeader extends Component {
     };
 
     render() {
-        const { classes, onBack, onClose, title } = this.props;
+        const { classes, onBack, onClose, title, isCategoryRoot } = this.props;
 
         return (
             <Fragment>
-                <Trigger key="backButton" action={onBack}>
-                    <Icon src={ArrowLeftIcon} />
-                </Trigger>
+                {!isCategoryRoot ? (
+                    <Trigger key="backButton" action={onBack}>
+                        <Icon src={ArrowLeftIcon} />
+                    </Trigger>
+                ) : (
+                    <div />
```

Comment:
As far as styling goes, I was expecting this `div` to simply slot into the space that the `Trigger` normally takes up. Is the container not set up properly for that? Can it be?

Reply:
@supernova-at : Actually removing the back arrow in specific case was distorting the UI completely so in order to match my work according to the shared mockups, I thought it would be better to add a div.

> As far as styling goes, I was expecting this div to simply slot into the space that the Trigger normally takes up. Is the container not set up properly for that? Can it be?

Sorry but I am unable to get what you meant by div to simply slot into the space that the trigger normally takes up.

As I investigated on trigger there is a portion occupied by button that contains svg of back arrow, in my work I thought to replace that with an empty div in order to meet the design changes.

This was my approach but as said earlier that I found no way to adjust UI other than this, please suggest. 

Thanks.

Reply:
Oh I just was hoping there was a way to do this without adding the extra `.isRoot` CSS class. No worries 👍